share_app/src/main/java/org/odk/share/activities/MainActivity.java

```diff
@@ -118,5 +120,9 @@ public void onLoaderReset(@NonNull Loader loader) {
     private void onItemClick(View view, int position) {
         Cursor cursor = formAdapter.getCursor();
         cursor.moveToPosition(position);
+        Intent intent  = new Intent(this, InstanceManagerTabs.class);
+        intent.putExtra(FORM_VERSION, cursor.getString(cursor.getColumnIndex(FormsProviderAPI.FormsColumns.JR_VERSION)));
```

Comment:
We already parse form id and version for the item. So let's reuse that instead of using cursor to fetch it

Reply:
How to reuse that? We can access only the `TextView` of the Item right?